pkg/scheduler/api/types.go

```diff
@@ -326,3 +326,14 @@ func (h HostPriorityList) Less(i, j int) bool {
 func (h HostPriorityList) Swap(i, j int) {
 	h[i], h[j] = h[j], h[i]
 }
+
+// ScheduleResult represents the result of one pod scheduled. It will contain
+// the final selected Node, along with the selected intermediate information.
+type ScheduleResult struct {
```

Comment:
Should this type be part of our `api` package at all? It seems like something that is only needed between `pkg/scheduler` and `pkg/scheduler/core`.

@bsalamat @k82cn